src/swap/SwapScreen.tsx

```diff
@@ -309,7 +309,6 @@ export function SwapScreen({ route }: Props) {
           fromTokenNetworkId: fromToken.networkId,
           amount: swapAmount[updatedField],
           amountType: swapAmountParam,
-          usdTotal: exchangeRate.swapAmount.multipliedBy(exchangeRate.price).toNumber(),
```

Comment:
I feel we have discussed this somewhere else, but should we calculate correctly this field instead of removing it?
I'm not sure if there is any value in keeping it though. So, I think it's fine to just remove it.

Reply:
Yes this is covered in `swap_execute_success` or `swap_execute_error`

https://github.com/valora-inc/wallet/blob/8e62e78216c2ce6bb5fac1b8db2c062474c06c19/src/analytics/Properties.tsx#L1358-L1359

which seems enough for now.